projects/gopacket/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/google/gopacket"
+primary_contact: "p.antoine@catenacyber.fr"
```

Comment:
Need to get approval from maintainers and then use them here as primary contact. move yourself to auto_ccs field.

Reply:
Ok